app/src/main/res/layout/home_fragment.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
general comment about all the fragment classes - I think it will be better pattern if we put all of them in a fragments package.